jdk/make/closed/autoconf/custom-hook.m4

```diff
@@ -1,5 +1,5 @@
 # ===========================================================================
-# (c) Copyright IBM Corp. 2017, 2024 All Rights Reserved
+# (c) Copyright IBM Corp. 2017, 2025 All Rights Reserved
```

Comment:
Changes to this file require running `bash common/autoconf/autogen.sh` to update `generated-configure.sh` files.

Reply:
Updated the files and removed the spaces in CopyToBuildJdk.gmk